frontend/src/pages/audio/_id/index.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <main :id="skipToContentTargetId" tabindex="-1">
-    <div v-if="backToSearchPath" class="w-full px-2 py-2 md:px-6">
+    <div v-if="backToSearchPath" class="w-full px-4 pb-4 md:px-8">
```

Comment:
@fcoveram, should the padding change starting at `md`, or at the `lg` breakpoint?

Reply:
@fcoveram the PR uses `md:`; is it correct, then? Could you review the PR when you can?

Reply:
I believe medium is correct, as in the PR.